policy/modules/services/container.te

```diff
@@ -414,12 +415,15 @@ corecmd_dontaudit_exec_all_executables(container_engine_domain)
 
 corenet_tcp_bind_generic_node(container_engine_domain)
 corenet_tcp_connect_http_port(container_engine_domain)
+corenet_tcp_connect_http_cache_port(container_engine_domain)
 corenet_tcp_bind_all_ports(container_engine_domain)
 corenet_udp_bind_all_ports(container_engine_domain)
 corenet_rw_tun_tap_dev(container_engine_domain)
 
 dev_getattr_all_blk_files(container_engine_domain)
 dev_getattr_all_chr_files(container_engine_domain)
+dev_getattr_generic_blk_files(container_engine_domain)
+dev_getattr_generic_chr_files(container_engine_domain)
```

Comment:
My immediate reaction to this was "doesn't `dev_getattr_all_{blk,chr}_files()` already grant this access? Turns out it doesn't because `device_t` doesn't have the `device_node` attribute. I'm wondering if maybe one of these interface pairs should be renamed.

Reply:
Ah that makes sense. I saw similar behavior at one point with `docker` but couldn't reproduce it.